R/tidiers.R

```diff
@@ -0,0 +1,209 @@
+#' @importFrom generics tidy glance augment
+#' @export
+generics::tidy
+#' @export
+generics::glance
+#' @export
+generics::augment
+
+#' @title Tidying methods for an mmrm model
+#' @description These methods tidy the estimates from an mmrm_tmb object into a
+#' summary
+#' @param x Fitted model object from the mmrm package
+#' @param conf.int If TRUE columns for the lower (conf.low) and upper bounds
+#'   (conf.high) of coefficient estimates are included., Default: FALSE
+#' @param conf.level Defines the range of the confidence internal.
+#'   Only used if conf.int = TRUE, Default: 0.95
+#' @param newdata data.frame, new data frame, Default: NULL
+#' @param se_fit logical, return standard errors of fit,
+#'   Default: interval!='none'
+#' @param interval type of interval calculation. Can be abbreviated,
+#'   Default: c("none", "confidence", "prediction")
+#' @param type.residuals Passed on to residuals.mmrm_tmb().
+#'   Default: c("response", "pearson", "normalized")
+#' @param \dots arguments passed on to predict method
+#' @return All tidying methods return a tibble.
+#'   The structure depends on the method chosen.
+#' @examples
+#' if(interactive()){
+#'
+#'   # fit a model
+#'     fit <- mmrm(FEV1 ~ RACE + SEX + ARMCD * AVISIT + us(AVISIT | USUBJID), data = fev_data)
+#'
+#'   # tidy method
+#'     fit |> tidy()
+#'     fit |> tidy(conf.int = TRUE)
+#'     fit |> tidy(conf.int = TRUE, conf.level = 0.9)
+#'
+#'   # glance method
+#'     fit |> glance()
+#'
+#'   # augment method
+#'     fit |> augment()
+#'     fit |> augment(interval = 'confidence')
+#'     fit |> augment(interval = 'prediction')
+#'
+#'     fit |> augment(type.residuals='pearson')
+#'     fit |> augment(type.residuals='normalized')
+#'  }
+#' @seealso
+#'  \code{\link[tibble]{as_tibble}}, \code{\link[tibble]{enframe}}
```

Comment:
Please use markdown format
```suggestion
#'  [tibble::as_tibble()], [tibble::enframe()]
```